stable/traefik/templates/deployment.yaml

```diff
@@ -61,6 +72,14 @@ spec:
       {{- if .Values.priorityClassName }}
       priorityClassName: {{ .Values.priorityClassName | quote }}
       {{- end }}
+      {{- if .Values.net_bind_service.enabled }}
```

Comment:
If `.Values.net_bind_service.enabled` is true and `.Values.securityContext` is set, we will have 2 blocks `securityContext` at the same level in `spec`. Is this block `securityContext` for NET_BIND_SERVICE related rather to `container` ?